webapp/channels/src/components/data_prefetch/data_prefetch.tsx

```diff
@@ -56,9 +56,12 @@ export default class DataPrefetch extends React.PureComponent<Props> {
 
     async componentDidUpdate(prevProps: Props) {
         const {currentChannelId, prefetchQueueObj, sidebarLoaded} = this.props;
+        if (sidebarLoaded && !prevProps.sidebarLoaded) {
+            await loadProfilesForSidebar();
```

Comment:
Can we remove this await? I don't really see why we need to wait for these to be loaded before we prefetch the other data.

If changing that breaks the unit tests because it doesn't have anything to wait on though, I think it's probably fine to leave this since it's just prefetching